Gruntfile.js

```diff
@@ -224,7 +225,8 @@ module.exports = function (grunt) {
             "uglify:" + target,
             "concat:" + target,
             "compress:" + target,
-            "clean:build"
+            "clean:build",
+            "clean:ciArtifacts"
```

Comment:
I'm not sure this is necessary as it should be taken care of by the "clean" call on line 219. Will any of the build steps trigger the tests to run?